pkgs/by-name/lc/lcevcdec/package.nix

```diff
@@ -95,6 +105,8 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.cmakeFeature "VN_SDK_FFMPEG_LIBS_PACKAGE" "")
     (lib.cmakeBool "VN_SDK_UNIT_TESTS" false)
     (lib.cmakeBool "VN_SDK_SAMPLE_SOURCE" false)
+    (lib.cmakeBool "VN_CORE_AVX2" stdenv.hostPlatform.avx2Support)
+    (lib.cmakeBool "VN_CORE_SSE" stdenv.hostPlatform.avx2Support)
```

Comment:
SSE support guarded behind avx2Support? Please explain.

Reply:
Should also be avx as that is required at runtime.